src/data/javascript-quiz.ts

```diff
@@ -1676,6 +1676,17 @@ const javascriptQuiz = [
       "?? is the optional nullish coalescing in JavaScript. It returns the first operand if it is not null or undefined. Otherwise, it returns the second operand.",
     Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator",
   },
+  {
+    Question:
+      "In JavaScript, which one gets higher precendence among 'setTimeout' and 'Promise' (considering zero delay)?",
+    Answer: "Promise",
+    Distractor1: "setTimeout",
+    Distractor2: "Depends on the order of calling",
+    Distractor3: "Unpredictable",
+    Explanation:
+      "Promise always has higher precendence than setTimeout. The Promise will be executed before the setTimeout, irrespective of the order in which they are called.",
+    Link: "https://dmitripavlutin.com/javascript-promises-settimeout/",
```

Comment:
please only use freeCodeCamp resources, official documentation or wikipedia articles for the link field

Reply:
Hey, sorry for linking an unofficial resource. To be honest it seemed to have some nice diagrams explaining the same, so linked it for the question. Will update the same.

Reply:
Updated, please have a look